main.go

```diff
@@ -37,10 +38,11 @@ var (
 	sc = &SafeConfig{
 		C: &Config{},
 	}
-	configFile    = flag.String("config.file", "blackbox.yml", "Blackbox exporter configuration file.")
-	listenAddress = flag.String("web.listen-address", ":9115", "The address to listen on for HTTP requests.")
-	showVersion   = flag.Bool("version", false, "Print version information.")
-	timeoutOffset = flag.Float64("timeout-offset", 0.5, "Offset to subtract from timeout in seconds.")
+
+	// Flags
+	configFile    string
```

Comment:
Could you use this style: https://github.com/alecthomas/kingpin#simple-example
rather than spreading each flag definition around?